recipes/xopen/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xopen" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "770d51ebb3b865e2ccaee05188e8a96cb959633de4456caa8d39ca52a878ca0f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
I am a bit confused by this:

1. This recipe was generated by conda skeleton, so I assumed it does a bit of best practice stuff
2. Isn't the `--no-deps` especially important so that any dependencies are installed by conda and __not__ by pip?

Reply:
Conda skeleton is not up to date, sorry
It is in my backlog to update it as well

Nowadays the "default" to install packages on ``conda-forge`` is using that way which I told you
https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---vv

Reply:
Thanks for the explanation!